subgraphs/stakedao/src/Oracle/index.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  Address,
+  BigDecimal,
+  BigInt,
+  dataSource,
+  log,
+} from "@graphprotocol/graph-ts";
+
+import * as constants from "./common/constants";
+import { getCurvePriceUsdc } from "./routers/CurveRouter";
+import { getPriceUsdc as getPriceUsdcUniswap } from "./routers/UniswapRouter";
+import { getPriceUsdc as getPriceUsdcSushiswap } from "./routers/SushiSwapRouter";
+
+export function getUsdPricePerToken(tokenAddr: Address): BigDecimal[] {
+  let network = dataSource.network();
+  let decimals = BigInt.fromI32(10)

```

Comment:
why not use a constant like `USDC_DENOMINATOR`?

Reply:
I am still not sure if this denominator will work for every `tokenAddr` or not, let's wait for others to test the implementation on their own subgraphs.